skipchain/skipchain.go

```diff
@@ -107,6 +112,15 @@ func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, on
 			return nil, onet.NewClientErrorCode(ErrorBlockContent,
 				"We're not responsible for latest block")
 		}
+		if len(prev.ForwardLink) > 0 {
+			return nil, onet.NewClientErrorCode(ErrorBlockContent,
+				"the latest block already has a follower")
+		}
+		if !s.newBlockStart(prev) {
+			return nil, onet.NewClientErrorCode(ErrorBlockInProgress,
+				"this skipchain-id is currently processing a block")
+		}
+		defer s.newBlockEnd(prev)
```

Comment:
Just to make sure: where is the check that there's no fork on the skipchain (i.e. same parent block for two distinct blocks), and is that check called after this defer ?